src/modules/AddressBook.js

```diff
@@ -0,0 +1,238 @@
+'use strict';
+
+var Utils = require('../qbUtils');
+var config = require('../qbConfig');
+
+/**
+ * Address Book
+ * @namespace QB.addressbook
+ */
+function AddressBook(service) {
+  this.service = service;
+}
+
+AddressBook.prototype = {
+  /**
+   * Method cud is create, update and delete contacts in address book. <br />
+   * An operation choosens by state: if contacts (to identify uses phone number) doesn't exist in address book it will save,
+   * if contact founds it will update, if passed a property `destroy` with 1 the contact will remove. <br />
+   * {@link https://quickblox.com/developers/AddressBook Found more here}. <br />
+   * The methods accepts 2 or 3 parameters. 
+   * @memberof QB.addressbook
+   * @param {Object[]} list - A list of contacts to create / update / delete.
+   * @param {Object} [options] - Search records with field which contains exactly specified value or by array of records' ids to retrieve
```

Comment:
I would suggest to remove this line, it's not clear what it is talking about. It's enough to have options.udid and tions.force only